public/jsx/components/card-icon.jsx

```diff
@@ -14,6 +14,7 @@ var CardIcon = React.createClass({
       'discover',
       'jcb',
       'maestro',
+      'mastercard',
       'master-card',
```

Comment:
Yeah, beacause the card validation returns master-card but the card type in the card list has MasterCard (which is then lower-cased). If that's from our end we can unify things in the service - (I didn't get around to checking) but since the mapping makes master-card into mastercard this will work fine.